packages/site-kit/base.css

```diff
@@ -566,3 +566,16 @@ input[type='checkbox']::after {
 input[type='checkbox']:checked::after {
 	left: calc(100% - 1em + 2px);
 }
+
+/* visually hidden, but accessible to assistive tech */
+.visually-hidden {
```

Comment:
I'm actually not much of a Svelte expert despite working on it a fair bit. Is there a reason to put this here instead of in `Permalink.svelte`?

Reply:
Yeah, I did this intentionally. `visually-hidden` is a common [utility class](https://piccalil.li/quick-tip/visually-hidden) used to hide content visually but still expose it to screen readers. It won't just be used in the Permalink component -- for instance, if I update the rendered markdown from the API to use that class, it will need to be available globally.

It could also be used in the future to hide other text, e.g. like hiding `<h1>`s on the [Svelte docs](https://github.com/sveltejs/svelte/blob/a39fc8de728ba292523828e1cf6f6254f125c640/site/src/routes/docs/index.svelte#L22).